lib/exercism/user_statistics.rb

```diff
@@ -0,0 +1,31 @@
+class UserStatistics
+  def self.of(username)
+    by_username(username)
+  end
+
+  def self.by_username(username)
+    user = User.find_by(username: username)
```

Comment:
Now we're finding the user twice because of the 404 in the action. How about passing the user to `of` instead of the username?